src/database/models.py

```diff
@@ -551,8 +570,8 @@ def full_json(self):
     communities = [cm.community.info() for cm in community_members]
     admin_at = [cm.community.info() for cm in CommunityMember.objects.filter(user=self, is_admin=True)]
     
-    data = model_to_dict(self, exclude=['real_estate_units',
-                                        'communities', 'roles'])
+    data = model_to_dict(self, exclude=['real_estate_units', 
+      'communities', 'roles', 'last_visited'])
```

Comment:
don't think we included last_visited.  Could fill with visit_log[-1]?

Reply:
Yeah, last_visited is not part of the model.